### ISSUE TITLE
Menu placement and position not working with slot "control"

### ISSUE THREAD

**Issue by QVllasa** [svelteuidev/svelteui#219]
### What package has an issue

@svelteuidev/core

### A clear and concise description of what the bug is

When I try using the menu component and set a custom trigger component via slot="control" I am not able to manage the placement and the position of the popper anymore.  

### In which browser(s) did the problem occur?

Chrome

### Steps To Reproduce

 ``` 
<Menu placement="{'end'}" position="{'bottom'}"   gutter={5}>
      <div slot="control" >Example Button</div>
      <Menu.Item on:click={() =>doStuff()}>Profile</Menu.Item>
      <Menu.Item on:click={() => doStuff()}>Logout</Menu.Item>
</Menu>
```

### Do you know how to fix the issue

No

### Are you willing to participate in fixing this issue and create a pull request with the fix

Yes

**Comment by BeeMargarida** [comment]
Hi, thank you for the bug report. This was already fixed in https://github.com/svelteuidev/svelteui/issues/215, but a new version has not been released yet, hopefully soon!

**Comment by BeeMargarida** [comment]
This should be fixed in the new version 0.7.6
